examples/hosts/app-integration/external-data/src/model/taskList.ts

```diff
@@ -66,55 +67,60 @@ export class TaskList extends DataObject implements ITaskList {
      * collection pattern -- see the contact-collection example for more details on this pattern.
      */
     private readonly tasks = new Map<string, Task>();
+    private _savedData: SharedMap | undefined;
```

Comment:
It would probably be worth adding comments to our 2 shared maps here to explain the difference between them / how we use them to handle external data syncing.